### ISSUE TITLE
Theme: color-background-* cannot be used because it is white

### ISSUE THREAD

**Issue by rphlwnk** [UI-Lovelace-Minimalist/UI#48]
I was using the `color-background-*` colors as a variable in my custom templates, but in the light mode all colors are set to white:

```
color-background-yellow : 250, 250, 250
color-background-blue : 250, 250, 250
color-background-green : 250, 250, 250
color-background-red : 250, 250, 250
```
When changing it to the same as in the dark mode, cards that use `icon_info_bg` are displayed wrong:
![image](https://user-images.githubusercontent.com/34039879/143719603-7da4fed2-f012-4b89-9998-90d250d2a17e.png) ![image](https://user-images.githubusercontent.com/34039879/143719611-d5902551-9548-4433-9d5d-f85a347fb317.png)

This is because the `opacity-bg` is set to 1, as soon as you set this to 0.1 (same as in the dark theme) everything is ok.

I dont know if this intended or not, i changed it on my environment and all other buttons are still looking smooth.

This relates to both themes:
- minimalist-mobile.yaml
- minimalist-desktop.yaml

**Comment by CM000n** [comment]
Well figured out! I will change it in the next release.

**Comment by CM000n** [comment]
Closed in v1.0.0-beta.8 🏳️‍🌈
